addons/l10n_ar_website_sale/demo/website_demo.xml

```diff
@@ -1,22 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo noupdate="1">
-
-    <record id="default_website_ri" model="website">
```

Comment:
Check with accounting team if it's okay to remove this and bellow product.pricelist demo data